### PR TITLE
Remove completed TODOs from code

### PR DESCRIPTION
**Pull request by md2119**
## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### CODE REVIEW

File: central/role/service/service_impl.go [hunk 1]
```diff
@@ -117,12 +117,6 @@ func (s *serviceImpl) CreateRole(ctx context.Context, roleRequest *v1.CreateRole
 	}
 	role.Name = roleRequest.GetName()
 
-	// Empty access scope ID is deprecated. Fill the default during the adoption
-	// period.
-	// TODO(ROX-9510): remove this block.
```
Comment (rukletsov) [inline central/role/service/service_impl.go:122]: This only partially resolves ROX-9510. Let's either address ROX-9510 completely or leave this out. I'm fine with any option.

File: central/graphql/resolvers/gen/main.go [hunk 2]
```diff
@@ -90,12 +90,6 @@ var (
 				ParentType: reflect.TypeOf(storage.Node{}),
 				FieldName:  "Scan",
 			},
-			// TODO(ROX-6194): Remove this entirely after the deprecation cycle started with the 55.0 release.
```
Comment (rukletsov) [inline central/graphql/resolvers/gen/main.go:93]: I'd suggest removing everything related to ROX-6194 in one PR and close the ticket. Can happen here or separately.

Comment (md2119) [reply central/graphql/resolvers/gen/main.go:93]: Everything related to ROX-6194 has already been removed except this,

```
message Policy {
...
reserved                                            10; //was repeated Exclusion whitelists
...
}
```

File: central/role/service/service_impl.go (hunk 1 above)
Comment (md2119) [reply central/role/service/service_impl.go:122]: Will revert this change and defer the changes to Authz team.

File: central/graphql/resolvers/gen/main.go (hunk 2 above)
Comment (rukletsov) [reply central/graphql/resolvers/gen/main.go:93]: I also see mentions in:
* `pkg/env/baseline.go`
* `qa-tests-backend/src/test/groovy/RiskTest.groovy`

**Pull request closed without merging**